app/js/components/LabResultEntry.jsx

```diff
@@ -230,7 +230,6 @@ export class LabResultEntry extends PureComponent {
     }
```

Comment:
What's going on in this file?

Reply:
Sorry, should have added that... when you add an order, you aren't (currently, at least) allowed to back-date it. So if a clerk is attempting to add results for a lab for which there is no order, and creates an order just to allow entry of the results, we want to allow the specimen date and the result date to be before the order date, so I removed the "min date" validation.

I also added the maxDate validation (can't be after the current date) to the result validation... not sure why that was missing before.